index.html

```diff
@@ -0,0 +1,169 @@
+          <i class="menuIcon material-icons">menu</i>
+          <i class="closeIcon material-icons">close</i>
+        </button>
+      </nav>
+    </header>
+    <main id="headline" class="headline">
+      <h1 class="title">A WEB DEVELOPMENT COURSE TO GET YOU STARTED</h1>
+      <p class="content">
+        Kickstart your courses in web development. Master html, css and javascript
+        courses using hands-on projects involving HTML, JavaScript, React.js, Ruby and more. 
+        No prior experience required.
+      </p>
+      <h2 class="date-set">2023.02.01(MON) - 15(THUR)</h2>
+      <p class="location">(Addis Ababa Arat Kilo Campus IT hall)</p>
+    </main>
+    <section class="main-program-list">
+      <h2 class="main-program-title">Main Program</h2>
+      <hr>
+      <section class="programs">
+          <section class="program-one">
+            <img class="program-tag" src="images/html.png" alt="Html logo">
+            <h2 class="program-title">HTML</h2>
+            <p class="program-content">Learn how to use CSS to style and visually orginize HTML pages</p>
+          </section>
+          <section class="program-two">
+            <img class="program-tag" src="images/css.png" alt="css logo">
+            <h2 class="program-title">CSS</h2>
+            <p class="program-content">Learn how to use CSS to style and visually orginize HTML pages</p>
+          </section>
+          <section class="program-three">
+            <img class="program-tag" src="images/js.png" alt="javascript logo">
+            <h2 class="program-title">JS</h2>
+            <p class="program-content">Learn how to use CSS to style and visually orginize HTML pages</p>
+          </section>
+      </section>
+      <button class="program-join-btn">Join the Class for 2023</button>
+      </section>
+
+      <h2 class="featured-speaker-title"></h2>
+
+      <section id="all-speakers">
+        
+      </section>
+      <div class="load-more">
+        <button onclick="myFunction()" class="my-btn" id="myBtn">Read more</button>
+      </div>
+     
+      
+    </section>
+
+    <section class="desktop-partner">
+      <div class="d-partner">
+        <h4 class="d-partner-title">Partner</h4>
+        <hr>
+        <ul class="d-sponsors">
+          <li class="d-mozilla">Mozilla</li>
+          <li class="d-google">Goole</li>
+          <li class="d-naver">NAVER</li>
+          <li class="d-airbnb">airbnb</li>
+          <li class="d-airoflot">Airoflot</li>
+        </ul>
+      </div>
+      <footer class="d-footer">
+        <div class="d-copy">@creative <br> Commons Kerea</div>
+        <div class="alright-reserved">
+          <p class="d-footer-content">Creative commons korea. Some rights reserved.</p>
+        </div>
+      </footer>
+    </section>
+  
+  <!-- About page section -->
+  <section id="about-course" class="about-page">
+    <div class="about-page-card">
+        <h2 class="about-page-title">
+          INPROVE YOUR LIFE THROUGH LEARNING
```

Comment:
- [ ] Kindly make sure to use `COCOGOOSE` font family for this title too to match the Behance design :recycle: